src/main/java/com/alekseij/todolist/service/EntryService.java

```diff
@@ -14,12 +15,15 @@ public EntryService(EntryRepository entryRepository) {
         this.entryRepository = entryRepository;
     }
 
+    public Entry getEntry(Long entryId) {
+        return entryRepository.findById(entryId).orElseThrow(() -> new IllegalStateException("Entry with id: " + entryId + " does not exist."));
+    }
+
     public List<Entry> getEntrys() {
         return entryRepository.findAll();
     }
 
     public void postNewEntry(Entry entry) {
```

Comment:
It is a good practice to return some data related to the newly create object in POST and PUT requests in REST API, as it will save client additional request to the REST API to fetch newly created resource. One option is to return generated id for Entity that will be stored in the DB. You can also return the newly created object, instead of just id